### PR TITLE
Add expiry date to albums

### LINKED ISSUE #125: User cannot purchase retired orders
As an authenticated user,
when I view a specific order,
when one order album is retired,
I should be able click link to view that album page,
but instead of 'add to cart' I see no longer available.
And the album no longer appears on any album index page.

### PR DESCRIPTION
**Pull request by scottfirestone**
Albums now have expiry dates. If an album is past is expiration date, the
album's show page will not show 'add to cart' but rather will show 'no longer
available'.

Albums have a default expiry date value of September 9, 2016.

Albums have 'expired?' method that returns true/false value.

The orders routes were changed from singular to plural; the custom orders/index
action was simply added to that resource.

closes #125 

@Draganovic @julsfelic @Salvi6God

### CODE REVIEW

**Comment by ghost** [comment]
Hey buddy, you got to fix the hound and merge issues!

File: app/views/orders/new.html.erb [hunk 1]
```diff
@@ -30,12 +30,13 @@
...
   </table>
 
-  <%= form_tag order_path, id: 'stripeForm' do %>
+  <%= form_tag @order, id: 'stripeForm' do %>
     <%= hidden_field_tag "total", @order.get_total %>
     <%= hidden_field_tag "stripeToken" %>
     <%= hidden_field_tag "stripeEmail" %>
     <%= hidden_field_tag "stripeAmount" %>
-    <%= button_to "Confirm Order", order_path(:order => {user_id: current_user.id, total: @order.get_total}), { id: "checkout" } %>
+    <%#= button_to "Confirm Order", order_path(@order, :order => {user_id: current_user.id, total: @order.get_total}), { id: "checkout" } %>
```
Comment (ghost) [inline app/views/orders/new.html.erb:38]: delete this comment

**Comment by ghost** [comment]
Cool, everything looks good!

**Comment by scottfirestone** [comment]
Thanks, merging!

**Pull request merged**